opensearch-operator/functionaltests/deploy_and_upgrade_test.go

```diff
@@ -101,3 +101,32 @@ var _ = Describe("DeployAndUpgrade", Ordered, func() {
 		Cleanup(name)
 	})
 })
+
+var _ = Describe("DeployWithHelm", Ordered, func() {
```

Comment:
Please add a comment here that the actual helm deployment happens outside the test code (otherwise it might be unclear where the cluster comes from).